src/renderer/feature/Login/selectors.ts

```diff
@@ -1 +0,0 @@
-import { RootState } from 'typesafe-actions';
```

Comment:
Removing this file requires fixing the import in `src/renderer/feature/Login/index.ts`.